examples/custom-pipeline/my-transport-process.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
This should be replace with `@vrbo/pino-rotating-file`

Reply:
We need to install the library to be able to share a example here...

is that okay?